orchestration/templates/cron-workflow.yaml

```diff
@@ -28,4 +22,4 @@ spec:
             - name: archive-path
               value: weekly_release/ClinVarVariationRelease_00-latest_weekly.xml.gz
             - name: gcs-prefix
-              value: '{{ printf "%s-%s-%sT%s:%s:%s" $YYYY $mm $dd $HH $MM $SS }}'
+              value: '{{ template "argo.timestamp" }}'
```

Comment:
Looks a lot neater with the timestamp code hidden away.

Reply:
I agree